### ISSUE TITLE
Creating a Translation Engine

### ISSUE THREAD

**Issue by amatefi** [paulfilkin/tradosClarity#15]
- on the target languages, if we added more than one, the screen reader doesn’t read the second or third language and the delete button for the language pair is not readable, is blank.
- add NMT models: When multiple target languages are present and we attempt to select all the models, the focus is lost. Additionally, two Add buttons are read by the screen reader, even though they serve different functions. To make a selection, the virtual cursor must be activated.
- Add the translation Memory --> The screen reader does not announce when the translation memory (TM) is selected, nor does it provide feedback after pressing the Add button. This issue is consistent across all 'Add' windows.
